ChatRPG/Pages/CampaignPage.razor.cs

```diff
@@ -48,9 +69,16 @@ protected override async Task OnInitializedAsync()
 
         _campaign = await PersisterService!.LoadFromCampaignIdAsync(
             CampaignMediatorService!.UserCampaignDict[_loggedInUsername!]);
+
         if (_campaign != null)
         {
-            _combatMode = _campaign.CombatMode;
+            _npcList = _campaign.Characters.ToList();
+            _currentLocation = _campaign.Environments.LastOrDefault();
+            _mainCharacter = _campaign.Player;
+            if (_campaign.CombatMode)
+            {
+                _activePromptType = PromptType.Attack;
```

Comment:
```suggestion
                _activePromptType = PromptType.Attack unless _campaign.Characters.ToList().first. pacifist?
```

Reply:
Ruby on rails invasion?! Make it stop!

Reply:
![image](https://github.com/KarmaKamikaze/ChatRPG/assets/74593817/91121dd7-f5bb-4101-9811-d8186beaec6c)